routes/customers/customersRoutes.js

```diff
@@ -27,8 +27,26 @@ router.get('/:id', (req, res) => {
 		}
 	})
 	.catch(err => {
-		res.status(500).json({ err: "Failed to retrieve the user" })
-	});
+		res.status(500).json({ err: "Failed to retrieve the user" });
+	})
+})
+
+router.get('/:email', (req, res) => {
+	const email = req.body.email;
```

Comment:
You're getting the email from the request body and not the request parameters so the `:email` parameter is unnecessary